apps/loader/chunked_upload.js

```diff
@@ -69,25 +69,22 @@ async function handleUpload(selectedFiles) {
     idtr.deleteCell(1);
   }
 
-  var currID = 0;
   // Add columns
-  for (var i=0; i<selectedFiles.length; i++, currID++) {
-    idtr.insertCell(-1).innerHTML = '<b>'+Number(currID+1)+'<b>';
-    fnametr.insertCell(-1).innerHTML = `<input type=text class="form-control" name=filename id='filename${currID}'
-    onchange=hideCheckButton();hidePostButton(); value='${selectedFiles[i]['name']}'>`;
-    tokentr.insertCell(-1).innerHTML = `<input type=text class="form-control" onchange=hideCheckButton();hidePostButton();
-    disabled name=token id='token${currID}'>`;
-    slidetr.insertCell(-1).innerHTML = `<input type=text class="form-control" name=slidename id='slidename${currID}'>`;
-    filtertr.insertCell(-1).innerHTML = `<input type=text class="form-control" name=filter id='filter${currID}'>`;
+  idtr.insertCell(-1).innerHTML = '<b>'+Number(1)+'<b>';
```

Comment:
I think we should get rid of the ID field entirely. At worst, it may confuse people that the new slide's uuid is just 1.

Reply:
Sure, I will do this.